test/unit-test/cellular_platform.h

```diff
@@ -0,0 +1,152 @@
+/*

```

Comment:
Why is this file needed? Where is the implementation?

Reply:
There will be some platform dependent code here, for cellular at command, the followings are needed:
#define Platform_Malloc    malloc
#define Platform_Free      free

And there will be other implementations updated with other test.